workspaces/api-gateway/src/api/v1/v1.module.ts

```diff
@@ -1,18 +1,49 @@
+import Stripe from 'stripe';
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
 import type { RouteTree } from '@nestjs/core';
 
 import { ProductsModule } from 'api/v1/products/products.module';
 import { CategoriesModule } from 'api/v1/categories/categories.module';
 import { V1Profile } from 'api/v1/v1.profile';
+import { PaymentMethod } from 'database/models/paymentMethod.entity';
+import { WebhooksController } from 'api/v1/webhook.controller';
+import { PaymentsModule } from 'api/v1/payments/payments.module';
+import { Payment } from 'database/models/payment.entity';
+import { StripeMethod } from 'database/models/paymentMethods/stripeMethod.entity';
+import { CreatePaymentService } from 'api/v1/payments/services/CreatePaymentService';
+import { CreatePaymentDetailService } from 'api/v1/payments/services/CreatePaymentDetailService';
+import { PaymentDetail } from 'database/models/paymentDetail.entity';
+import { CodMethod } from 'database/models/paymentMethods/codMethod.entity';
 
 @Module({
-  imports: [ProductsModule, CategoriesModule],
-  providers: [V1Profile],
+  imports: [
+    TypeOrmModule.forFeature([
+      PaymentMethod,
+      Payment,
+      PaymentDetail,
+      CodMethod,
+      StripeMethod,
+    ]),
```

Comment:
why import this one here? Why don't we import for the module that use PaymentMethod/Payment/...?